addOns/wappalyzer/wappalyzer.gradle.kts

```diff
@@ -38,5 +38,7 @@ dependencies {
     implementation("org.apache.xmlgraphics:batik-gvt:$batikVersion")
     implementation("org.apache.xmlgraphics:batik-util:$batikVersion")
 
+    implementation("org.jsoup:jsoup:1.13.1")
```

Comment:
Is this really needed? The `WappalyzerPassiveScanner` is already using Jericho to parse the contents.

Reply:
Jericho lacks jquery-like selector capabilities that's why I used Jsoup.

Reply:
OK, this should be mentioned in the commit message, otherwise going forward one has to wonder why the add-on is using two parsing libraries.

There's `jericho-selector` library that provides that, although does not seem to be maintained anymore.

Reply:
OK, I have updated the commit message.

Reply:
None of the commit messages mention that yet.

Reply:
I will add it to the latest commit